docs/source/transforms.rst

```diff
@@ -154,11 +158,13 @@ that relies on the :class:`torch.utils.data.DataLoader` with ``num_workers >
 
 Transforms tend to be sensitive to the input strides / memory layout. Some
 transforms will be faster with channels-first images while others prefer
-channels-last. You may want to experiment a bit if you're chasing the very
-best performance. Using :func:`torch.compile` on individual transforms may
-also help factoring out the memory layout variable (e.g. on
+channels-last. Like ``torch`` operators, most transforms will preserve the
+layout of the input, but this may not always be respected due to implementation
+details.  You may want to experiment a bit if you're chasing the very best
+performance.  Using :func:`torch.compile` on individual transforms may also help
+factoring out the memory layout variable (e.g. on
 :class:`~torchvision.transforms.v2.Normalize`). Note that we're talking about
-**memory layout**, not tensor shape.
+**memory layout**, not :ref:`tensor shape <conventions>`.
```

Comment:
It is called memory format and memory layout is about strided tensor or sparse variants: https://pytorch.org/docs/stable/tensor_attributes.html#torch-layout